### ISSUE TITLE
Build in Andriod studio?

### ISSUE THREAD

**Issue by amirmaaref** [staf621/ki4a#38]
Hi, Would you please tell me how to build and compile after clone it in Android Studio? thanks!

**Comment by ashishpujari204** [comment]
1) Open Android Studio
2) Click on Open an existing Android Studio project
3) Select your project from directory.
4) After project selection studio will compile your project.

**Comment by Mr-Philipp** [comment]
I am also not able to open this project nor to import it. For me it looks like .idea file is broken or misconfigured.

**Comment by Belval** [comment]
Shameless plug, but I needed to rebuild the project for my personal use. I reconfigured the project with Android Studio and gradle in the process. Feel free to use that.

https://github.com/Belval/ki4a

**Comment by mohsensaremi** [comment]
@Belval Can you change the package name and build and connect to VPN successfully?
After changing package name connection wont be established.

**Comment by Belval** [comment]
No, as you noted in another issue, the executables are hardcoded with the package name. I tried to do a string replace with `sed` but it did not work.
